doc/authentication.md

```diff
@@ -50,22 +127,53 @@ BoxAPIConnection api = new BoxAPIConnection("YOUR-CLIENT-ID",
     "YOUR-CLIENT-SECRET", "YOUR-ACCESS-TOKEN", "YOUR-REFRESH-TOKEN");
 ```
 
-Auto-Refresh
-------------
+As User
+-------
+
+The purpose of as user is to be used by enterprise administrators to make API calls on behalf of their managed users. 
+This can also be used by a Service Account to make API calls for managed users or app users. 
 
-By default, a `BoxAPIConnection` will automatically refresh the access token if
-it has expired. To disable auto-refresh, set the connection's auto-refresh
-setting to false with [`setAutoRefresh(false)`][auto-refresh]. Keep in mind that
-you will have to manually refresh the access token yourself.
+In order to invoke as user calls you can use 
 
 ```java
-// This connection won't auto-refresh.
-BoxAPIConnection api = new BoxAPIConnection("YOUR-CLIENT-ID",
-    "YOUR-CLIENT-SECRET", "YOUR-ACCESS-TOKEN", "YOUR-REFRESH-TOKEN");
-api.setAutoRefresh(false);
+BoxAPIConnection api = new BoxAPIConnection("YOUR-ACCESS-TOKEN");
+api.asUser("USER-ID");
+```
+
+once you are done making calls on behalf of a managed user or app user you can switch back to the admin or service account with
+
+```java
+api.asSelf();
+```
 
-// If the access token expires, you will have to manually refresh it.
-api.refresh();
+Token Exchange
+--------------
+
+You can exchange a client's access token for one with a lower scope, in order to restrict the permissions for a child 
+client or to pass to a less secure location (e.g. a browser-based app). This is useful if you want to use the 
```

Comment:
The word "client" throughout this paragraph should probably be changed to "API connection" to reflect Java SDK terminology